exams/pearson.py

```diff
@@ -141,6 +149,19 @@ def _profile_country_to_alpha3(profile):
 ], field_prefix='profile')
 
 
+write_ead_file = _tsv_writer([
+    ('AuthorizationTransactionType', 'operation'),
+    ('ClientAuthorizationID', 'id'),
+    ('ClientCandidateID', 'user.profile.student_id'),
+    ('ExamSeriesCode', 'course.program.exam_series_code'),
+    ('Modules', 'course.exam_module'),
+    ('Accommodations', lambda _: ''),
```

Comment:
Strings here are treated as property paths, so `''` would currently trigger an error. Pearson wanted us to add this column, but we don't have a defined use case for it yet, so it gets left blank.  `_writer` takes either a string property path or a callable here hence the lambda returning an empty string. An alternative would be to update `_writer` to recognize `None` values here, then the csv writer would convert `None` to `''`.